CMakeLists.txt

```diff
@@ -109,8 +109,17 @@ if (BUILD_SDF)
   ign_find_package(ignition-utils1 VERSION REQUIRED)
   set(IGN_UTILS_VER ${ignition-utils1_VERSION_MAJOR})
 
+  ########################################
+  # Find ignition common
+  ign_find_package(ignition-common4 COMPONENTS graphics REQUIRED_BY usd)
+  set(IGN_COMMON_VER ${ignition-common4_VERSION_MAJOR})
+
+  ########################################
+  # Find PXR
+  ign_find_package(pxr QUIET REQUIRED_BY usd PKGCONFIG pxr)
```

Comment:
Looks like CI has a configuration warning still.

Reply:
Configuration warnings should be gone now thanks to https://github.com/ignition-tooling/release-tools/pull/625, but I realize that until this PR is merged, https://github.com/ignition-tooling/release-tools/pull/625 will cause CI warnings on other sdformat PRs, as mentioned in https://github.com/ignition-tooling/release-tools/pull/625#issuecomment-1017018168. I'm not sure if we should merge this PR first to take care of CI warnings on other PRs, or revert https://github.com/ignition-tooling/release-tools/pull/625 until it's time to merge this PR.

Reply:
Do you think we can merge this today? If so, I think it's fine, but if not, there are at least two PRs waiting to be merged blocked on the CI failure, so I would recommend reverting the release-tools change.

Reply:
Yeah, I think this can be merged today if the fix I mentioned in https://github.com/ignitionrobotics/sdformat/pull/817#discussion_r790107049 seems valid. The only lingering question I have is about if we want to structure include guards in a particular way for the SDF -> USD and USD -> SDF converters, which isn't really a blocker: https://github.com/ignitionrobotics/sdformat/pull/817#discussion_r791011920

If CI is green, everything else has been addressed, so I think we should be good to go.

Reply:
I addressed the include guard fix, so I'm just waiting on CI to see if it comes back green :crossed_fingers: